src/main/java/com/team1816/lib/hardware/YamlConfig.java

```diff
@@ -81,8 +81,14 @@ public String toString() {
 
         public static SubsystemConfig merge(SubsystemConfig active, SubsystemConfig base) {
             var result = new SubsystemConfig();
-
-            result.implemented = active.implemented || base.implemented;
+            if(active.implemented = true)
```

Comment:
Compile error: use `==` instead of `=` for checking equality.

Also, the change does not implement the desired functionality - it is essentially still the same as before, just more verbose.

Your code should check if `active.implemented` _exists_ (is declared), and that value should take precedence over base.implemented regardless of whether it is true or false. If it's not declared, then it should use base.implemented.